lib/_http_client.js

```diff
@@ -297,6 +310,24 @@ function ClientRequest(input, options, cb) {
 Object.setPrototypeOf(ClientRequest.prototype, OutgoingMessage.prototype);
 Object.setPrototypeOf(ClientRequest, OutgoingMessage);
 
+Object.defineProperty(ClientRequest.prototype, 'aborted', {
+  get() {
+    return this[kWritableState].destroyed;
+  },
+  set(val) {
+    this[kWritableState].destroyed = val;
+  }
+});
+
+Object.defineProperty(ClientRequest.prototype, 'destroyed', {
+  get() {
+    return this[kWritableState].destroyed;
+  },
+  set(val) {
+    this[kWritableState].destroyed = val;
```

Comment:
TODO for later... `destroyed` should likely be read only to user code

Reply:
I added the getter because it exists in `Writable` (for compat reasons). Though thinking about it. Adding it here is not required per se. Should I remove it?

Reply:
removed